doc/languages-frameworks/gnome.section.md

```diff
@@ -40,12 +40,6 @@ Unlike other libraries mentioned in this section, GdkPixbuf only supports a sing
 
 `gdk-pixbuf` contains [a setup hook](#ssec-gnome-hooks-gdk-pixbuf) that sets `GDK_PIXBUF_MODULE_FILE` from dependencies but as mentioned in further section, it is pretty limited. Loaders should propagate this setup hook.
 
-### Icons {#ssec-gnome-icons}
```

Comment:
We definitely want to retain this section, since it explains how icon themes work. We will want to change `pass it to our wrapper` to something like ``add their datadirs to `XDG_ICON_DIRS` environment variable (this is Nixpgks specific, not actually a XDG standard variable)``.

Reply:
Done

Reply:
I wonder if we should include a paragrapn like:

````
In the rare case you need to use icons from dependencies (e.g. when an app forces an icon theme), you can use the following to pick them up:

```nix
  buildInputs = [
    pantheon.elementary-icon-theme
  ];
  preFixup = ''
    gappsWrapperArgs+=(--prefix XDG_DATA_DIRS : "$XDG_ICON_DIRS")
  '';
```
````

Reply:
Done

Reply:
Could you please also move that last change into a separate paragraph, rather than putting it in the middle of the first one? That is, something like:

````patch
-When an application uses icons, an icon theme should be available in `XDG_DATA_DIRS` during runtime. The package for the default, icon-less [hicolor-icon-theme](https://www.freedesktop.org/wiki/Software/icon-theme/) (should be propagated by every icon theme) contains [a setup hook](#ssec-gnome-hooks-hicolor-icon-theme) that will pick up icon themes from `buildInputs` and add their datadirs to `XDG_ICON_DIRS` environment variable (this is Nixpgks specific, not actually a XDG standard variable). In the rare case you need to use icons from dependencies (e.g. when an app forces an icon theme), you can use the following to pick them up:
+When an application uses icons, an icon theme should be available in `XDG_DATA_DIRS` during runtime. The package for the default, icon-less [hicolor-icon-theme](https://www.freedesktop.org/wiki/Software/icon-theme/) (should be propagated by every icon theme) contains [a setup hook](#ssec-gnome-hooks-hicolor-icon-theme) that will pick up icon themes from `buildInputs` and add their datadirs to `XDG_ICON_DIRS` environment variable (this is Nixpgks specific, not actually a XDG standard variable). Unfortunately, relying on that would mean every user has to download the theme included in the package expression no matter their preference. For that reason, we leave the installation of icon theme on the user. If you use one of the desktop environments, you probably already have an icon theme installed.
+
+In the rare case you need to use icons from dependencies (e.g. when an app forces an icon theme), you can use the following to pick them up:
 
 ```nix
   buildInputs = [
     pantheon.elementary-icon-theme
   ];
   preFixup = ''
     gappsWrapperArgs+=(--prefix XDG_DATA_DIRS : "$XDG_ICON_DIRS")
   '';
 ```
 
-Unfortunately, relying on that would mean every user has to download the theme included in the package expression no matter their preference. For that reason, we leave the installation of icon theme on the user. If you use one of the desktop environments, you probably already have an icon theme installed.
````

Reply:
As far as I understand that, `relying on that` in the original text meant `relying on wrapper`, so having that text after the rare-case example preserves the meaning while moving that just after XDG_ICON_DIRS text makes it feel like relying on XDG_ICON_DIRS makes every user to download the theme :thinking:

Reply:
Yeah, it meant `relying on wrapper (specifically, on picking up an icon theme from buildInputs into the wrapper)`. Currently it is ordered 1. describe hook 2. mention rare exception 3. mention why it is not used as one might expect. Going from general to specifice (i.e. swapping 2. and 3. makes more sense to me). Though I agree that the wording could be better.

Reply:
Done